Code/Framework/AzFramework/AzFramework/Windowing/NativeWindow.cpp

```diff
@@ -90,82 +97,170 @@ namespace AzFramework
 
     bool NativeWindow::IsActive() const
     {
-        return m_pimpl != nullptr && m_pimpl->IsActive();
+        if (m_pimpl != nullptr)
+        {
+            return m_pimpl->IsActive();
+        }
+        else
+        {
+            return false;
+        }
     }
 
     void NativeWindow::SetWindowTitle(const AZStd::string& title)
     {
-        m_pimpl->SetWindowTitle(title);
+        if (m_pimpl != nullptr)
+        {
+            m_pimpl->SetWindowTitle(title);
+        }
     }
 
     WindowSize NativeWindow::GetClientAreaSize() const
     {
-        return m_pimpl->GetClientAreaSize();
+        if (m_pimpl != nullptr)
+        {
+            return m_pimpl->GetClientAreaSize();
+        }
+        else
+        {
+            return WindowSize();
+        }
     }
 
     WindowSize NativeWindow::GetMaximumClientAreaSize() const
     {
-        return m_pimpl->GetMaximumClientAreaSize();
+        if (m_pimpl != nullptr)
+        {
+            return m_pimpl->GetMaximumClientAreaSize();
+        }
+        else
+        {
+            return WindowSize();
+        }
```

Comment:
repeat: This can be made into a ternary statement
```suggestion
        return m_pimpl != nullptr ? m_pimpl->GetMaximumClientAreaSize() : WindowSize{};
```